### PR TITLE
Fix some compiler warnings

### PR DESCRIPTION
**Pull request by seven1240**
fix some compiler warnings on debian 10

### CODE REVIEW

**Comment by seven1240** [comment]
fixed err like this


```
http_parser.c: In function ‘http_method_d’:
http_parser.c:467:13: warning: this ‘if’ clause does not guard... [-Wmisleading-indentation]
             if (MATCH(s, "PUT")) code = http_method_put; break;
             ^~
http_parser.c:467:58: note: ...this statement, but the latter is misleadingly indented as if it were guarded by the ‘if’
             if (MATCH(s, "PUT")) code = http_method_put; break;
                                                          ^~~~~
http_parser.c:466:13: warning: this ‘else’ clause does not guard... [-Wmisleading-indentation]
             else
             ^~~~
http_parser.c:467:58: note: ...this statement, but the latter is misleadingly indented as if it were guarded by the ‘else’
             if (MATCH(s, "PUT")) code = http_method_put; break;
                                                          ^~~~~

```

File: libsofia-sip-ua/msg/msg_date.c [hunk 1]
```diff
@@ -248,7 +248,8 @@ issize_t msg_date_d(char const **ss, msg_time_t *date)
     /* actime-date =
        wkday SP month SP ( 2DIGIT | ( SP 1DIGIT )) SP time SP 4DIGIT */
     mon = month_d(s); skip_token(&s);
-    if (mon < 0 || !IS_LWS(*s)) return -1; s++;
+    if (mon < 0 || !IS_LWS(*s)) return -1;
+    s++;
```
Comment (crienzo) [inline libsofia-sip-ua/msg/msg_date.c:252]: This one is interesting- I assume the `s++` is correct and the compiler didn't uncover a bug?

Comment (crienzo) [reply libsofia-sip-ua/msg/msg_date.c:252]: Looks OK- the `while (IS_LWS(*s)) s++;` below suggests this is correct and was simply confusing whitespace.

**Comment by andywolk** [comment]
Duplicate of #80

**Pull request closed without merging**